avocado/utils/download.py

```diff
@@ -44,12 +44,18 @@ def url_open(url, data=None, timeout=5):
     old_timeout = socket.getdefaulttimeout()
     socket.setdefaulttimeout(timeout)
     try:
+        if proxies and isinstance(proxies, dict):
```

Comment:
Yep, you can actually supply `None` so you can simply call `handler = urllib2.ProxyHandler(proxies)`